### PR TITLE
LPS-167846 [CP] Error when filtering a specific Environment Type

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/Header/hooks/useFilters.js [hunk 1]
```diff
@@ -117,6 +117,17 @@ export default function useFilters(setFilterTerm, productName) {
...
+					if (
+						environmentType === 'Production' &&
+						!filters.environmentTypes.value.includes(
+							'Non-Production'
+						)
+					) {
+						return `${accumulatorEnvironmentTypesFilter}${
+							index > 0 ? ' or ' : ''
+						}contains(productName, '${environmentType}') and not contains(productName, 'Non-Production')`;
+					}
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/Header/hooks/useFilters.js:129]: Note: @jairmedeiros this was a tricky suggested by Amos to unblock the issue with filter

This should be taken out someday

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**